python/tests/plugins/test_gen_json.py

```diff
@@ -0,0 +1,66 @@
+from unittest import TestCase
+from tempfile import TemporaryDirectory, NamedTemporaryFile
+
+from aac.plugins.gen_json.gen_json_impl import print_json
+from aac.plugins.plugin_execution import PluginExecutionStatusCode
+
+
+class Testgen_json(TestCase):
```

Comment:
[Python code style conventions](https://www.python.org/dev/peps/pep-0008/#class-names) dictate classes use CamelCase style. 
```suggestion
class TestGenJson(TestCase):
```